internal/engine/sqlite/convert.go

```diff
@@ -325,6 +354,10 @@ func (c *cc) convertMultiSelect_stmtContext(n *parser.Select_stmtContext) ast.No
 	}
 }
 
+func (c *cc) convertExprListContext(n *parser.Expr_listContext) ast.Node {
+	return c.convertExprLists(n.AllExpr())
```

Comment:
I think you are returning a nested "ast.List" here, but an un-nested "ast.List" would be correct.
```suggestion
	return c.convertExprLists(n.AllExpr()).Items[0]
```

Reply:
https://www.sqlite.org/syntax/expr.html

Reply:
Updated in https://github.com/kyleconroy/sqlc/pull/2274/commits/2adb18f5b8898ab8d67f3fe924e7d86ea3eb7c10

Reply:
@hakobera thanks!